### PR TITLE
Add partial capture support

### PR DESCRIPTION
**Pull request by AnasNaouchi**

### CODE REVIEW

**Review by ajzkk — APPROVED** [review]

File: operations/charge.go [hunk 1]
```diff
@@ -51,18 +51,19 @@ func (req *ListCharges) Describe() *internal.Description {
...
+	Amount                   int64                  	`json:"amount"`
+	Currency                 string                 	`json:"currency"`
+	Offsite                  omise.OffsiteTypes     	`json:"offsite,omitempty"`
+	Description              string                 	`json:"description,omitempty"`
+	DontCapture              bool                   	`json:"-"` // inverse, since `capture` defaults to true
+	ReturnURI                string                 	`json:"return_uri,omitempty"`
+	Metadata                 map[string]interface{} 	`json:"metadata,omitempty"`
+	ExpiresAt                *time.Time             	`json:"expires_at,omitempty"`
+	ZeroInterestInstallments *bool                  	`json:"zero_interest_installments,omitempty"`
+	AuthorizationType		 omise.AuthorizationType	`json:"authorization_type,omitempty"`
```
Comment (aashishgurung) [inline operations/charge.go:66]: Indentation between field, type and tag doesn't look right.

Comment (overlab-io) [reply operations/charge.go:66]: try viewing in vscode it shows properly

Comment (aashishgurung) [reply operations/charge.go:66]: While it's true that certain editors may provide automatic formatting and display improvements, it's important for us to maintain code readability and consistency across different environments and editors.

Comment (overlab-io) [reply operations/charge.go]: Fixed

**Review by aashishgurung — APPROVED** [review]

**Pull request merged**